AstraEngine.Core/Entity.cs

```diff
@@ -8,6 +8,10 @@ namespace AstraEngine.Core;
 /// </summary>
 public class Entity
 {
+    /// <summary>Name for the entity</summary>
+    public string Name { get; set; } = "";

```

Comment:
Microsoft guidelines suggest using `string.Empty` rather than `""`.  See: https://learn.microsoft.com/en-us/dotnet/api/system.string.empty?view=net-8.0